OBSDOCS-219 - Child PR - Loki Queries (#65235)
openshift/openshift-docs: modules/logging-loki-logcli.adoc

// Module is included in the following assemblies:
// 
//

> :_content-type: REFERENCE
> [id="logging-loki-logcli-about_{context}"]
> 
> = Querying Loki

Reviewer: ```suggestion
:_content-type: REFERENCE
[id="logging-loki-logcli-about_{context}"]
= Querying Loki
```

> You can use Loki's command-line interface `logcli` to query logs.

Reviewer: ```suggestion
You can use the Loki `logcli` CLI to query logs.
```
CLI doesn't need to be spelled out, CLI is used throughout OCP docs. See https://docs.openshift.com/container-platform/4.13/cli_reference/index.html

Can we add a bit more context here / use cases? Query logs for what? What does querying logs provide the user with?

> .Example Application Log Query

Reviewer: ```suggestion
.Example application log query
```
Use sentence case for all headings

[source,terminal]
----
$ oc extract cm/lokistack-sample-ca-bundle --to=lokistack --confirm
$ cat lokistack/*.crt >lokistack_ca.crt

> $ logcli -o raw --bearer-token="${bearer_token}" --ca-cert="lokistack_ca.crt" --addr xxxxxx

Reviewer: One command per code block. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

![image](https://github.com/openshift/openshift-docs/assets/40172997/6449191e-f8b3-46cb-b54e-7e62f4bc3cba)

----

.Example Infrastructure Log Query
[source,terminal]
----
$ logcli --bearer-token="$(oc whoami -t)" --addr https://lokistack-dev-openshift-logging.apps.devcluster.openshift.com/api/logs/v1/infrastructure labels
----

.Example Audit log Query
[source,terminal]
----
$ logcli --bearer-token="$(oc whoami -t)" --addr https://lokistack-dev-openshift-logging.apps.devcluster.openshift.com/api/logs/v1/audit labels
----

> .Additional Resources

Reviewer: See guidance for ID for additional resources: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

> .Additional Resources

Reviewer: ```suggestion
[role="_additional-resources"]
.Additional resources
```
